pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -22,6 +22,9 @@ const (
 	// AnnotationBuildRefSecret is an annotation that tells the Build Controller to reconcile on
 	// events of the secret only if is referenced by a Build in the same namespace
 	AnnotationBuildRefSecret = "build.build.dev/referenced.secret"
+
+	// AnnotationBuildRefRepository is a label key that tells if the Build Controller will check remote repository exists or not
```

Comment:
This comment seems to ambiguous. Does this annotation denote that the remote repository 'has been' verified, or 'should be' verified?

Reply:
Also, why would we not just always check if it exists?

Reply:
@coreydaley Thanks for your review!
> This comment seems to ambiguous. Does this annotation denote that the remote repository 'has been' verified, or 'should be' verified?
>
I have modified comments to be more clear, pls take a look again, thanks!
>Also, why would we not just always check if it exists?
>
I think this comment can help you understand why don't I check url alway: https://github.com/shipwright-io/build/issues/464#issuecomment-726222903 . If you still have some concern, pls let me know, thanks!

Reply:
This should be ok by now. @coreydaley pls resolve if you agree